recipes/pytest-watch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pytest-watch" %}
+{% set version = "4.1.0" %}
+{% set sha256 = "29941f6ff74e6d85cc0796434a5cbc27ebe51e91ed24fd0757fad5cc6fd3d491" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # Skip build on win32 untill we have a solution for conda-build issue #2619
+  skip: True  # [win32]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pytest-watch = pytest_watch:main
+    - ptw = pytest_watch:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest >= 2.6.4
```

Comment:
You need to remove the space between the logical operator and the version number. E.g.:

```
    - pytest >=2.6.4
```